Healthy/Classes/Extensions/UITextField+Style.swift

```diff
@@ -18,40 +18,34 @@ extension UITextField {
 
         layer.cornerRadius = Constants.defaultCornerRadius
         layer.borderWidth = Constants.defaultBorderWidth
-        layer.borderColor = Constants.defaultBorderColor.cgColor
+        layer.borderColor = UIColor.gray4.cgColor
         borderStyle = .none
+    }
+}
+
+// MARK: Constants
 
-        // Workaround for adding leading & training spaces
-        setLeftPaddingPoints()
-        setRightPaddingPoints()
+private extension UITextField {
+    enum Constants {
+        static let defaultCornerRadius: CGFloat = 10.0
+        static let defaultHeight: CGFloat = 55.0
+        static let defaultPadding: CGFloat = 12.0
+        static let defaultBorderWidth: CGFloat = 1.5
     }
 }
 
 // MARK: set right / left padding to text field
 
 extension UITextField {
```

Comment:
I'm against workaround, can we use auto-layout spacing in FormTextField instead?